app/views/pafs_core/projects/steps/confidence_homes_by_gateway_four.html.erb

```diff
@@ -4,26 +4,28 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <%= pafs_form_for @project, url: { action: "save", id: @project.to_param, step: @project.step } do |f| %>
+
         <%= render partial: "pafs_core/error_summary", locals: { f: f } %>
-        <h1 class="govuk-heading-l"><%= t('.heading') %></h1>
-        <div class="step-form">
-          <div class ="form-group">
-            <%= f.hidden_field :id %>
-            <fieldset>
-              <legend class="govuk-visually-hidden"><%= t(".heading") %></legend>
+
+        <%= f.hidden_field :id %>
+
+        <%= f.govuk_fieldset legend: { text: t('.heading'), size: 'l' } do %>
+          <%= form_group(@project.step, :confidence_homes_by_gateway_four) do %>
+            <legend class="govuk-visually-hidden"><%= t(".heading") %></legend>
+
+            <%= f.govuk_radio_buttons_fieldset(:confidence_homes_by_gateway_four, legend: nil) do %>
               <% PafsCore::Confidence::CONFIDENCE_VALUES.each do |value| %>
-                <%= f.radio_button :confidence_homes_by_gateway_four, 
-                      value,
-                      label: confidence_band_label(:homes_by_gateway_four, value) 
+                <%= f.govuk_radio_button :confidence_homes_by_gateway_four, 
```

Comment:
So the `govuk_error_summary` links to the first radio correctly you'd need to add `link_errors: true` to the first radio. Alternatively you could use a [govuk_collection_radio_buttons](https://govuk-form-builder.netlify.app/form-elements/radios/#radios-collection-with-legend-and-hint-text) here instead.

Reply:
Thanks @peteryates, I hadn't noticed the `link_errors` option.

Reply:
No worries @PaulDoyle-DEFRA - I wish it wasn't necessary but the builder can't "detect" which the first radio button is so it needs to be told. Glad to see your implementation's going really well!